django/applications/catmaid/control/log.py

```diff
@@ -24,10 +25,10 @@ def log_frontent_event(request, level='info'):
         logger = logging.getLogger('catmaid.frontend')
         entry = "User: %s (%s) %s" % (
             request.user.username, request.user.id, msg)
-        result = log(logger, level, entry)
```

Comment:
log() didn't return anything, and adding the type annotation exposed this. Fortunately, nothing was using result anyhow